app/views/alchemy/admin/partials/_upload_button.html.erb

```diff
@@ -0,0 +1,31 @@
+<%= form_for [alchemy, :admin, form_object], html: {multipart: true, class: 'upload-button'} do |f| %>
```

Comment:
Let's keep it as-is, if people need this to be used outside of Alchemy, let's add that complexity later.